lock-impl/src/main/java/com/palantir/lock/impl/LockServiceImpl.java

```diff
@@ -478,6 +471,20 @@ private void tryLocks(LockClient client, LockRequest request, BlockingMode block
         }
     }
 
+    private void logAtDebugOrInfo(String lockId, LockClient currentHolder, long duration) {
+        if (log.isDebugEnabled()) {
+            log.debug("Blocked for {} ms to acquire lock {} {}.",
+                    duration,
+                    lockId,
+                    currentHolder == null ? "successfully" : "unsuccessfully");
+        } else {
+            log.info("Blocked for {} ms to acquire lock {} {}.",
```

Comment:
I am happy to discuss having a separate appender but am worried that every product will have to configure it. Not sure how it helps if it is done automatically. I can just name the logger differently, if we want slow logs to be more easily filtered.